package.json

```diff
@@ -117,5 +117,9 @@
   },
   "browserslist": [
     "extends @instructure/browserslist-config-instui"
-  ]
+  ],
+  "dependencies": {
+    "marked-react": "^2.0.0",
+    "uuid": "^9.0.1"
+  }
```

Comment:
is there any reason why these need to be in the root package.json?

Reply:
Nope :D